docs/GETTING-STARTED.md

```diff
@@ -13,6 +13,9 @@ const options = {
     },
   },
   apis: ['./routes.js'], // Path to the API docs
+  jsDocFilter: (jsDocComment) => { // Optional filtering mechanism applied on each API doc
```

Comment:
To be honest, it feels a bit weird to return a boolean after receiving an argument. I mean, if this is meant to be a flag for a logic to be attached, I'd personally prefer something like `jsDocFilter: true` and pass the function on another property of the definition object. Or maybe simply attach the function which returns a reduced object.